### DISCUSSION TITLE
Show Class Names in Navigator

### DISCUSSION THREAD

**Discussion by mmiroshnikov** [webstudio-is/webstudio-community#165] [Wishlist]
It is much easier to navigate (and — even more importantly — have an incentive to name classes properly) when the class names are visible in the Navigator.
See a WF example

<img width="357" alt="Screenshot 2024-07-16 at 09 57 31" src="https://github.com/user-attachments/assets/7c5c69f2-c087-4faa-bb10-38f548a7b8a0">

**Comment by kof** [comment]
Token name and instance name might/will be not the same. We considered this.

**Reply by mmiroshnikov** [reply]
> Token name and instance name might/will be not the same. We considered this.

Thank you.

Now instead of a clearly named structure I have Box > Box > Box > Box > Box...
